build.gradle

```diff
@@ -258,13 +259,14 @@ publishMods {
 		announcementTitle = "<:curseforge:900697838453936149> Download from CurseForge"
 		projectSlug = "skyblocker"
 		requires("fabric-api")
-		optional("roughly-enough-items", "emi")
+		optional("roughly-enough-items", "emi", "jei")
+		embeds("yacl")
 	}
 
 	discord {
 		webhookUrl = System.getenv("DISCORD_WEBHOOK")
 		username = "Changelog"
-		content = changelog.map { "<@&1134565945482948638>\n## ${displayName}\n" + it}
+		content = changelog.map { "<@&1134565945482948638>\n## Skyblocker v${mod_version} for ${minecraft_version}\n" + it}
```

Comment:
I guess the `v` here is intentional?

Reply:
Yeah, when I last asked it was to help separate the mod version from the Minecraft version.